doc.go

```diff
@@ -56,6 +56,17 @@ type ObjectName struct {
 	Name string `json:"object"`
 }
 
+// Workload name
+//
+// swagger:parameters workloadDetails
+type DeploymentParam struct {
```

Comment:
Is this type "DeploymentParam" valid ?

Reply:
yep, it is used by swagger to generate the documentation for the URL. I just changed the struct name to WorkloadParam.